src/components/GlobalLinterMessages/index.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import LinterMessage from '../LinterMessage';
+import {
+  fakeExternalLinterMessage,
+  createFakeLocation,
+  createContextWithFakeRouter,
+} from '../../test-helpers';
+import { createInternalMessage } from '../../reducers/linter';
+import { getCodeLineAnchorID } from '../CodeView/utils';
+
+import GlobalLinterMessages, { PublicProps } from '.';
+
+describe(__filename, () => {
+  const render = (props: PublicProps) => {
+    return shallow(<GlobalLinterMessages {...props} />);
+  };
+
+  const renderWithMount = (props: PublicProps) => {
+    const location = createFakeLocation();
+    const context = createContextWithFakeRouter({ location });
+    return mount(<GlobalLinterMessages {...props} />, context);
+  };
+
+  it('renders no linter messages', () => {
+    const root = render({ messages: [] });
```

Comment:
This needs one more test: `render({ messages: undefined })`. That's an important difference because an empty array and an undefined variable will be handled differently.